group_vars/all.yml

```diff
@@ -80,6 +80,7 @@ scale:
 
 #extra_templates:
 
+#heat_configs:
```

Comment:
We can use the parameters defaults in https://github.com/redhat-performance/jetpack/blob/master/group_vars/all.yml to add Tripleo Heat Templates configuration options